test/native/jest.config.js

```diff
@@ -18,6 +18,19 @@ const transpiledPackageNames = glob( 'packages/*/src/index.{js,ts}' ).map(
 	( fileName ) => fileName.split( '/' )[ 1 ]
 );
 
+// The following unit tests related to the `raw-handling` API will be enabled when addressing
+// the various errors we encounter when running them in the native version.
+// Reference: https://github.com/WordPress/gutenberg/issues/55652
+const RAW_HANDLING_UNSUPPORTED_UNIT_TESTS = [
+	'html-formatting-remover',
+	'phrasing-content-reducer',
+	'ms-list-converter',
+	'figure-content-reducer',
+	'special-comment-converter',
+	'normalise-blocks',
+	'image-corrector',
+];
```

Comment:
We'll enable these unit tests when addressing https://github.com/WordPress/gutenberg/issues/55652.